src/plugins/wizard/public/saved_visualizations/_saved_vis.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  createSavedObjectClass,
+  SavedObjectOpenSearchDashboardsServices,
+} from '../../../saved_objects/public';
+import { EDIT_PATH, PLUGIN_ID, WIZARD_SAVED_OBJECT } from '../../common';
+
+export function createSavedWizardVisClass(services: SavedObjectOpenSearchDashboardsServices) {
+  const SavedObjectClass = createSavedObjectClass(services);
+
+  class SavedWizardVis extends SavedObjectClass {
+    public static type = WIZARD_SAVED_OBJECT;
+
+    // if type:wizard has no mapping, we push this mapping into OpenSearch
+    public static mapping = {
```

Comment:
it's important thing to note that i believe once this gets merged in. if stuff was removed (i believe) or fields type was changed, a migration will need to be created.